src/compiler/transpile.ts

```diff
@@ -1,14 +1,40 @@
 import { rollupPluginUtils } from '@compiler-deps';
+import {
+  Config,
+  TransformCssToEsmInput,
+  TransformOptions,
+  TranspileOptions,
+  TranspileResults,
+} from '@stencil/core/internal';
```

Comment:
Is changing this import what is causing the the need to update `scripts/bundles/compiler.ts`?

Reply:
not exactly, the change is necessary because now the generated `d.ts` file for `src/compiler/index.ts` depends on a type exported from `src/compiler/transpile.ts` (so in terms of build output `build/compiler/index.d.ts` depends on `build/compiler/transpile.d.ts`)

Reply:
the underlying issue is that typescript doesn't provide any functionality for bundling together `.d.ts` files and I haven't found a good solution for doing that automatically yet, so until and if we find a good way to do that when we add a type export from a module like this we'll have to make a change in the build scripts to copy the `.d.ts` over as well.